include/libdnf/conf/vars.hpp

```diff
@@ -34,19 +34,43 @@ namespace libdnf {
 /// of directories.
 struct Vars {
 public:
+    enum class Priority { DEFAULT = 10, AUTO = 20, FILE = 30, ENVIRONMENT = 40, COMMANDLINE = 50, RUNTIME = 60 };
```

Comment:
Replace `FILE` with `CONFIG`? It would probably make a better sense from a user perspective.
I looked at Option::Priority and I'm wondering if `EMPTY` and `PLUGIN` wouldn't make sense in some cases.

Reply:
I was thinking about `EMPTY` variables. We have a fixed set of configuration options. But the variables behave differently. At the beginning we have only three variables `arch`,` basearch`, `releasever`. The user can add more variables. Instead of creating an `EMPTY` variable, the user does not create the variable at all. Do we want to set an existing variable to `EMPTY`? Isn't it better to introduce variable removal support?

`FILE` <-> `CONFIG` and `PLUGIN`. For discussion.

Reply:
PLUGIN priority is also available for config but it is unused. With variables I would prefer that plugin that want to modify variables will create a file in `/etc/dnf/vars` and the best would be when the file will be in RPM. Therefore I would prefer to not add `PLUGIN` priority at least from beginning.
With `FILE` <-> `CONFIG` I thing that the source is more like `VARS_DIRS` (It is a list option with "/etc/yum/vars", "/etc/dnf/vars")